config_file.py

```diff
@@ -223,6 +226,7 @@
 # common default values.  The canvas, board, and template colors from
 # http://paletton.com/#uid=50F0u0kllllaFw0g0qFqFg0-obq
 COMMON_VALS = {'version': 'NONE',
+               'last_path': '\\pyrouterjig.png',
```

Comment:
On a Mac, this results in the default filename `\pyrouterjig0.png` in the current working directory. Removing the `\\` fixed the issue.